Content.Server/_RMC14/Rules/CMDistressSignalRuleSystem.cs

```diff
@@ -763,6 +768,7 @@ private void CheckRoundShouldEnd()
 
             var marines = EntityQueryEnumerator<ActorComponent, MarineComponent, MobStateComponent, TransformComponent>();
             var marinesAlive = false;
+            var marineList = new List<EntityUid>();
```

Comment:
This should be a field on the system that gets cleared before iterating the query, for performance